### PR TITLE
Fixes gpgkey locator value to a new one used in 6.2

### PR DESCRIPTION
**Pull request by oshtaier**
+bonus - fixed data used for tests according to fixed BZs
```
nosetests tests/foreman/ui/test_gpgkey.py
.....E.SS....SSS.....S....S.S....S.S....
======================================================================
ERROR: Create gpg key with valid name and valid gpg key via file import then fail to update its name
----------------------------------------------------------------------
Ran 40 tests in 4565.478s

FAILED (SKIP=10, errors=1)

nosetests tests/foreman/ui/test_gpgkey.py -m test_negative_update_name_for_imported_content
.
----------------------------------------------------------------------
Ran 1 test in 372.806s

OK
```

### CODE REVIEW

**Comment by oshtaier** [comment]
Locator was fixed before my PR, but I decided to submit my PR anyway as it has few good improvements

**Comment by sthirugn** [comment]
@oshtaier can you please update commit message?

File: robottelo/ui/gpgkey.py [hunk 1]
```diff
@@ -22,29 +22,23 @@ def _search_locator(self):
...
+            self.wait_until_element(
+                locators['gpgkey.file_path']).send_keys(key_path)
+        elif key_content:
+            self.click(locators['gpgkey.content'])
+            self.wait_until_element(
+                locators['gpgkey.content']).send_keys(key_content)
         else:
             raise UIError(
-                'Could not create new gpg key "{0}"'.format(name)
+                u'Could not create new gpgkey "{0}" without contents'
```
Comment (elyezer) [inline robottelo/ui/gpgkey.py]: thanks for prefixing `u`

**Comment by elyezer** [comment]
ACK, pending @sthirugn suggestion.

File: robottelo/ui/gpgkey.py (hunk 1 above)
Comment (oshtaier) [inline robottelo/ui/gpgkey.py]: It was there actually, I removed one redundant nested condition

**Comment by oshtaier** [comment]
@sthirugn Done

**Comment by elyezer** [comment]
Looks good to me, @sthirugn should merge it after a second look.

**Comment by sthirugn** [comment]
ACK

**Pull request merged**